cloudtasks/flex_app/main.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"golang.org/x/net/context"
+
+	"cloud.google.com/go/compute/metadata"
+	"cloud.google.com/go/datastore"
+)
+
+var (
+	datastoreClient *datastore.Client
+
+	payloadKey = datastore.NameKey("CloudTaskPayload", "Singleton", nil)
+)
+
+type Payload struct {
+	Body string
+}
+
+func main() {
+	ctx := context.Background()
+	var err error
+
+	projectID := os.Getenv("GCLOUD_PROJECT")
+	if projectID == "" {
+		projectID, err = metadata.ProjectID()
+		if err != nil {
+			log.Fatalf("Could not get project ID: %v", err)
+		}
+	}
+
+	datastoreClient, err = datastore.NewClient(ctx, projectID)
+	if err != nil {
+		log.Fatalf("Could not initialize datastore client: %v", err)
+	}
+
+	http.HandleFunc("/", rootHandle)
+	http.HandleFunc("/payload", payloadHandle)
```

Comment:
We performed user studies and there was a bit misunderstanding about the string "log_payload".
Please replace "payload" with "example_task_handler" in this sample.
The Python master samples have been changed: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1288/files

Reply:
do we need the word "example"? why not just "task_handler"